### PR TITLE
Flying docs

### PR DESCRIPTION
**Pull request by JWittmeyer**
main PR: https://github.com/code-kern-ai/cognition-ui/pull/39

### CODE REVIEW

File: controller/transfer/cognition/minioUpload.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+from controller.task_queue import manager as task_queue_manager
+
+
+def handle_cognition_file_upload(path_parts: List[str]):
+
+    if path_parts[1] != "_cognition":
+        return
+
+    if path_parts[3] == "chat_tmp_files":
+        if path_parts[5] == "queued":
```
Comment (FelixKirschKern) [inline controller/transfer/cognition/minioUpload.py]: What do you think of combining the conditions as we're not distinguishing between both states further?

File: controller/transfer/cognition/minioUpload.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+                conversation_item.created_by,
+                {
+                    "cognition_project_id": cognition_project_id,
+                    "conversation_id": conversation_id,
+                    "minio_path": "/".join(path_parts[1:]),
+                    "bucket": path_parts[0],
+                },
+                True,  # not sure if prio is right here as the prio tasks should only take < 1 min but waiting for the normal queue will take ages depending on the queue
+            )
+        return
```
Comment (FelixKirschKern) [inline controller/transfer/cognition/minioUpload.py]: no need for the return here

File: controller/transfer/cognition/minioUpload.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
+from typing import List
```
Comment (FelixKirschKern) [inline controller/transfer/cognition/minioUpload.py]: file name is in camelCase

File: controller/transfer/cognition/minioUpload.py (hunk 1 above)
Comment (JWittmeyer) [reply controller/transfer/cognition/minioUpload.py]: was initially planned with multiple stages, not the case anymore => combined

**Review by FelixKirschKern — APPROVED** [review]

**Pull request merged**